wire-ios-data-model/Source/Model/QualifiedID.swift

```diff
@@ -56,7 +56,7 @@ public extension ZMConversation {
         guard
             let context = managedObjectContext,
             let uuid = remoteIdentifier,
-            let domain = domain ?? ZMUser.selfUser(in: context).domain
+            let domain = domain ?? ZMUser.selfUser(in: context).domain ?? BackendInfo.domain
```

Comment:
I think checking the self user is redundant as it should be the same as checking BackendInfo.

Reply:
@johnxnguyen so back to what it was?

Reply:
I was thinking more 

```swift
let domain = domain ?? BackendInfo.domain
```